portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -41,8 +48,14 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     // sends user input to doGet
     doGet(request,response);
-    response.sendRedirect("/index.html");
 
+    // adds comments to datastore 
+    String comment = request.getParameter("comments");
+    Entity taskEntity = new Entity("Task");
+    taskEntity.setProperty("comments", comment);
```

Comment:
consider setting "comments" as a constant at class-level:

e.g. public static final String TASK_COMMENT_PROPERTY = "comment";

Since you will also need to reference it to get the comments.

Reply:
I got a lot of errors when I tried to add that and replace my "comments" with it.